src/components/NavBar/NavBar.vue

```diff
@@ -88,7 +143,26 @@ export default {
       isLoading: false,
       searchModel: null,
       search: null,
-      searchQueue: []
+      searchQueue: [],
+      notifItems: [
+        { avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg', message: 'Ali Connors I\'ll be in your neighborhood doing errands this weekend. Do you want to hang out?', date: '3/12/2021, 7:14:33 PM', readed: false },
```

Comment:
try using back ticks and `${}` to inject dynamic data into the string

Reply:
I am using a new data field call subject for rendering
![image](https://user-images.githubusercontent.com/37958655/158076774-22842d31-3fd2-49a2-878a-d53f8a949951.png)
![image](https://user-images.githubusercontent.com/37958655/158076784-af8c98ce-a729-43f7-802c-ca7d10e5d2cb.png)